webpack.config.babel.js

```diff
@@ -56,7 +56,8 @@ const config = {
     },
     output: {
         filename: 'bundle.js',
-        path: path.resolve(__dirname, 'build', 'client')
+        path: path.resolve(__dirname, 'build', 'client'),
+        publicPath: "localhost:8080/dev-server/bundle.js"
```

Comment:
`"/dev-server"` would be enough. You shouldn't be specifying the entire scheme here.